hbase-thrift/src/main/java/org/apache/hadoop/hbase/thrift2/client/ThriftAdmin.java

```diff
@@ -1157,6 +1158,18 @@ public boolean isSnapshotCleanupEnabled() {
     throw new NotImplementedException("isSnapshotCleanupEnabled not supported in ThriftAdmin");
   }
 
+  @Override
+  public List<SlowLogPayload> getSlowLogResponses(final List<ServerName> serverNames)
+      throws IOException {
+    throw new NotImplementedException("getSlowLogResponses not supported in ThriftAdmin");
```

Comment:
Why not? We adding thrift serialization support in a separate patch?

Reply:
I think we should support it in the thrift interface... we support a thrift interface, don't we? Or is it deprecated? I don't know the current state of thrift.

Reply:
Ya, I had a similar question about thrift interface when I was working on HBASE-23330, but it looks like it is supported. So we should probably do this too.. it is not too much code right? (just defining the equivalent thrift specs).